### ISSUE TITLE
problems following instructions on vignette

### ISSUE THREAD

**Issue by nicholasjhorton** [beanumber/airlines#54]
I'm running the github versions of etl and airlines.

When I ran:

library(airlines)
db <- src_mysql(host="localhost", user="root", password="benb", dbname="airlines")
ontime <- etl("airlines", db = db, dir = "~/dumps/airlines")
ontime %>%
  etl_init() %>%
  etl_update(years = 2015:2016)

I got the following error message when it got to 2016_06:

trying URL 'http://tsdata.bts.gov/PREZIP/On_Time_On_Time_Performance_2016_5.zip'
Content type 'application/x-zip-compressed' length 23829775 bytes (22.7 MB)
==================================================
downloaded 22.7 MB

downloaded 0 bytes

Error in (function (url, destfile, method, quiet = FALSE, mode = "w",  : 
  cannot download all files
In addition: Warning message:
In (function (url, destfile, method, quiet = FALSE, mode = "w",  :
  URL 'http://tsdata.bts.gov/PREZIP/On_Time_On_Time_Performance_2016_6.zip': status was '404 Not Found'

I'm on a Mac.

**Comment by nicholasjhorton** [comment]
ontime %>%
  etl_init() %>%
  etl_update(years = 2013:2015)
ontime %>%
  etl_update(years=2016, months=1:5)

worked for me.

**Comment by beanumber** [comment]
Yes, the June file doesn't exist and currently it does not fail gracefully.
